src/coreComponents/physicsSolvers/NonlinearSolverParameters.cpp

```diff
@@ -215,29 +215,38 @@ NonlinearSolverParameters::NonlinearSolverParameters( string const & name,
+                        viewKeysStruct::timeStepIncreaseIterLimString() <<
+                        "should be smaller than " << viewKeysStruct::timeStepDecreaseIterLimString(),
+                        getWrapperDataContext( viewKeysStruct::timeStepIncreaseIterLimString() ) );
 
   GEOS_ERROR_IF_LE_MSG( m_lineSearchResidualFactor, 0.0,
-                        getWrapperDataContext( viewKeysStruct::lineSearchResidualFactorString() ) << ": should be positive" );
+                        viewKeysStruct::lineSearchResidualFactorString() << "should be positive",
+                        getWrapperDataContext( viewKeysStruct::lineSearchResidualFactorString() ) );
 
   if( m_oscillationScaling > 0 )
   {
     // check oscillation parameters
     GEOS_ERROR_IF_LE_MSG( m_oscillationScalingFactor, 0.0,
-                          getWrapperDataContext( viewKeysStruct::oscillationScalingFactorString() ) << ": should be positive" );
+                          viewKeysStruct::oscillationScalingFactorString() << "should be positive",
+                          getWrapperDataContext( viewKeysStruct::oscillationScalingFactorString()) );
     GEOS_ERROR_IF_GT_MSG( m_oscillationScalingFactor, 1.0,
-                          getWrapperDataContext( viewKeysStruct::oscillationScalingFactorString() ) << ": can not be more than 1.0" );
+                          viewKeysStruct::oscillationScalingFactorString() << "can not be more than 1.0",
+                          getWrapperDataContext( viewKeysStruct::oscillationScalingFactorString()) );
     GEOS_ERROR_IF_LT_MSG( m_oscillationCheckDepth, 2,
-                          getWrapperDataContext( viewKeysStruct::oscillationCheckDepthString() ) << ": can not be less than 2" );
+                          viewKeysStruct::oscillationCheckDepthString() << "can not be less than 2",
+                          getWrapperDataContext( viewKeysStruct::oscillationCheckDepthString()) );
     GEOS_ERROR_IF_LE_MSG( m_oscillationTolerance, 0.0,
-                          getWrapperDataContext( viewKeysStruct::oscillationToleranceString() ) << ": should be positive" );
+                          viewKeysStruct::oscillationToleranceString() << "should be positive",
+                          getWrapperDataContext( viewKeysStruct::oscillationToleranceString()) );
     GEOS_ERROR_IF_GE_MSG( m_oscillationTolerance, 1.0,
-                          getWrapperDataContext( viewKeysStruct::oscillationToleranceString() ) << ": can not be more than 1.0" );
+                          viewKeysStruct::oscillationToleranceString() << "can not be more than 1.0",
+                          getWrapperDataContext( viewKeysStruct::oscillationToleranceString()) );
     GEOS_ERROR_IF_LT_MSG( m_oscillationFraction, 0.0,
-                          getWrapperDataContext( viewKeysStruct::oscillationFractionString() ) << ": can not be negative" );
+                          viewKeysStruct::oscillationFractionString() << "can not be negative",
```

Comment:
missing spaces in this file
```suggestion
                          viewKeysStruct::oscillationFractionString() << "can not be negative",
```